WordPress/Sniffs/WP/AlternativeFunctionsSniff.php

```diff
@@ -126,11 +126,37 @@ public function getGroups() {
 				),
 			),
 
-			'file_system_read' => array(
+			'unlink' => array(
 				'type'      => 'warning',
-				'message'   => 'File operations should use WP_Filesystem methods instead of direct PHP filesystem calls. Found: %s()',
+				'message'   => '%s() is discouraged. Use wp_delete_file() to delete a file.',
+				'since'     => '4.2.0',
+				'functions' => array(
+					'unlink',
+				),
+			),
+
+			'rename' => array(
+				'type'      => 'warning',
+				'message'   => '%s() is discouraged. Use WP_Filesystem::move() to rename a file.',
+				'since'     => '2.5.0',
+				'functions' => array(
+					'rename',
+				),
+			),
+
+			'file_system_operations' => array(
+				'type'      => 'warning',
+				'message'   => 'File operations should use WP_Filesystem methods instead of direct PHP filesystem calls. Found: %s().',
 				'since'     => '2.5.0',
 				'functions' => array(
+					'chgrp',
+					'chmod',
+					'chown',
+					'is_writable',
+					'is_writeable',
+					'mkdir',
+					'rmdir',
+					'touch',
```

Comment:
While these new function names are in alphabetical order, could you make the whole list alphabetical please? That'll make it easy to read in the future.

Reply:
@GaryJones I'm fine with merging this as it is - I will sort out the alphabetization later. I have PR to update this sniff lined up as part of the PHPCSUtils/modern PHP PR series anyway, so I'll just do it in that PR.

Reply:
(and yes, noticed it myself too and already put it on my to-do list)

Reply:
👍🏻